src/components/PageWithSideMenu.tsx

```diff
@@ -55,19 +56,20 @@ export const SideNavHeader = ({ icon, text, id }: SideNavHeaderProps) => {
 };
 
 export const NavigationList = ({ sx, ...props }: BoxProps) => (
-  <nav>
-    <Box
-      sx={{
-        pt: '0.5rem',
-        display: 'flex',
-        flexDirection: 'column',
-        gap: '0.5rem',
-        a: { textTransform: 'none' },
-        ...sx,
-      }}
-      {...props}
-    />
-  </nav>
+  <Box
+    component="nav"
+    sx={{
+      pb: '0.5rem',
+      mx: '0.5rem',
+      display: 'flex',
+      maxWidth: '75%',
```

Comment:
Personlig syns jeg det blir kunstig lange knapper i mange tilfeller, og at det hadde vært penere om den knappen som trenger mest plass bestemmer lengden, men det kan du og Cec enes om så er det greit tenker jeg :D
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/dd3f1f69-b3a8-4aa7-9b5c-7de1b8a5849b)


En annen ting å merke seg her er at i de tilfellene knappen trenger mer plass vil den bli over to linjer i stedet for å bruke ekstra ledig plass til høyre 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/f57ae5ab-6eea-4cad-a80a-d15b158feebf)

Reply:
Avklart med Cecilie at lengden blir satt til lengste knapp,